website/docs/working-with-containers/registries/index.md

```diff
@@ -7,13 +7,38 @@ keywords: [podman desktop, podman, containers, registries]
 hide_table_of_contents: false
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Preconfigured from './\_preconfigured-registry.md'
+import Custom from './\_custom-registry.md'
+import Verification from './\_verification-private-registry.md'
+
 # Working with container registries
 
 Podman Desktop has access to public registries by default.
 
-You might want to configure Podman Desktop to:
+You might want to configure Podman Desktop to authenticate to:
```

Comment:
`You may want to` instead? Unsure if this is correct grammar

Reply:
Style guide says: use "might" or "can" rather than "may":

https://github.com/redhat-documentation/vale-at-red-hat/blob/3854352b74fe762665c69e00b4ce5caa9dd06992/.vale/styles/RedHat/TermsWarnings.yml#L48C8-L48C13